chainer/links/model/vision/resnet.py

```diff
@@ -548,13 +548,14 @@ class BuildingBlock(link.Chain):
 
```

Comment:
Can you add the n_layer to the doc string and notify that it is deprecated?
Otherwise the interface changes wont be reflected on the docs.

Reply:
Oh, sorry but I forgot it. I will update a document. :pray:

Reply:
Updated docstring in 9c7b8fe.

Reply:
Thanks!!